poster/styles/styles.scss

```diff
@@ -11,480 +7,451 @@ sans-serif;
+    grid-gap: 0;
+    color: #FAFAF2;
 }
 
-.body-linethirtythree {
-    margin-left: 8vw;
+/*title location within outer wrapper*/
+.tophalf {
+    grid-row: 1/2;
 }
 
-.body-linethirtyfour {
-    display: inline-block;
-    margin-top: 2vw;
-    margin-left: 18vw;
-}
+/*bottom half grid establishing 2 columns and text properties*/
+.bottomhalf {
+    grid-row: 2/3;
+    grid-column: 1/2;
+    display: grid;
+    grid-template-columns: repeat(7, 45vw);
+    grid-template-rows: 1fr;
+    line-height: 2vw;
+    padding-left: 2vw;
+    font-family: Arial, Helvetica, sans-serif;
+    line-height: 2.2vw;
 
-.body-linethirtyfive {
-    margin-left: 16vw;
-}
+    h2 {
+        font-size: 2.6vw;
+        font-weight: bold;
+    }
 
-.body-linethirtysix {
-    margin-left: 13.6vw;
-}
+    div,
+    h3,
+    p,
+    li {
+        font-size: 1.7vw;
+        font-weight: 200;
+    }
 
-.body-linethirtyseven {
-    margin-left: 10.8vw;
-}
+    ul {
+        list-style-type: none;
+    }
 
-.body-linethirtyeight {
-    margin-left: 8.4vw;
 }
 
-.body-linethirtynine {
-    margin-left: 6vw;
-}
+/*left and right margins for body (set in div tags)*/
+.line__ {
+    &one {
+        margin-top: 6vw;
+    }
 
-.body-lineforty {
-    margin-left: 4vw;
-}
+    &two {
+        margin-left: 4vw;
+    }
 
-.body-linefortyone {
-    margin-left: 2vw;
-}
+    &three {
+        margin-left: 8vw;
+    }
 
-.body-linefortythree {
-    display: block;
-    margin-top: -4.3vw;
-    margin-left: 30vw;
-    margin-bottom: -2vw;
-}
+    &four {
+        margin-top: 2vw;
+        margin-left: 30vw;
+    }
 
-.body-linefortyfour {
-    margin-left: 28vw;
-}
+    &five {
+        margin-top: -2vw;
+        margin-left: 16vw;
+    }
 
-.body-linefortyfive {
-    margin-left: 25vw;
-}
+    &six {
+        margin-left: 4vw;
+    }
 
-.body-linefortysix {
-    margin-left: 21vw;
-}
+    &eight {
+        margin-left: 4vw;
+    }
 
-.body-linefortyseven {
-    margin-left: 18vw;
-}
+    &ten {
+        margin-left: 12vw;
+    }
 
-.body-linefortyeight {
-    margin-left: 15vw;
-}
+    &eleven {
+        margin-left: 8vw;
+    }
 
-.body-linefortynine {
-    margin-left: 11vw;
-}
+    &twelve {
+        margin-left: 12vw;
+        margin-bottom: -2vw;
+    }
 
-.body-linefifty {
-    margin-left: 8vw;
-}
+    &thirteen {
+        margin-top: -1.5vw;
+        margin-left: 12vw;
+    }
 
-.body-linefiftyone {
-    margin-left: 4vw;
-}
+    &fourteen {
+        margin-left: 8vw;
+    }
 
-.body-linefiftytwo {
-    margin-left: 8vw;
-}
+    &fifteen {
+        margin-left: 4vw;
+    }
 
-/*display block to move up like in poster*/
+    &sixteen {
+        margin-left: 8vw;
+    }
 
-.body-linefiftythree {
-    display: block;
-    font-size: .7vw;
-    margin-left: 36vw;
-    margin-top: -3.5vw;
-}
+    &eighteen {
+        margin-left: 4vw;
+    }
 
-/*bottom half grid establishing 2 columns and text properties*/
-.bottomhalf {
-    grid-row: 2/3;
-    grid-column: 1/2;
-    margin-top: 0;
```

Comment:
A `div`, by default, doesn't have a top margin, so this rule shouldn't be necessary.